platforms/android/library-compose/src/main/java/io/element/android/wysiwyg/compose/RichTextEditorState.kt

```diff
@@ -22,7 +22,19 @@ import uniffi.wysiwyg_composer.MenuAction
  * multiple [RichTextEditor] composables.
  */
 @Stable
-class RichTextEditorState internal constructor() {
+class RichTextEditorState internal constructor(
+    initialHtml: String = "",
+) {
+    /**
+     * The content of the editor as HTML formatted for sending as a message.
+     */
+    var messageHtml by mutableStateOf("")
```

Comment:
`var messageHtml by mutableStateOf(initialHtml)` ? (removing the `init{}` block)